setup.py

```diff
@@ -63,96 +49,29 @@ def run(self):
 
 
 setup(
-    name='natcap.invest',
-    description="InVEST Ecosystem Service models",
-    long_description=README,
-    maintainer='James Douglass',
-    maintainer_email='jdouglass@stanford.edu',
-    url='http://github.com/natcap/invest',
-    packages=find_namespace_packages('src'),
-    package_dir={'': 'src'},
-    include_package_data=True,
     install_requires=_REQUIREMENTS,
-    python_requires='>=3.8,<3.11',
-    license='BSD',
-    long_description_content_type='text/x-rst',
-    zip_safe=False,
```

Comment:
This kwarg is obsolete. https://setuptools.pypa.io/en/latest/deprecated/zip_safe.html?highlight=zip_safe